LnkParse3/decorators.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+from datetime import timezone
+from struct import unpack
+import functools
+from LnkParse3.lnk_exception import lnk_exception
+
+
+def must_be(expected):
+    def outer(func):
+        @functools.wraps(func)
+        def inner(self, *args, **kwargs):
+            result = func(self, *args, **kwargs)
+
+            # FIXME: delete
+            return result
+
+            if result != expected:
+                error = "%s must be %s: %s" % (func.__name__, expected, result)
+                raise lnk_exception(error)
+
+            return result
+
+        return inner
+
+    return outer
+
+
+def uuid(func):
+    @functools.wraps(func)
+    def inner(self, *args, **kwargs):
+        binary = func(self, *args, **kwargs)
+
+        # FIXME: delete
+        return binary.hex()
```

Comment:
I would "change the way of processing" in the separate commit because of tests, otherwise, I see no problem.

Reply:
Yes, I will change `hex()` to Microsoft GUID, a variant of UUID, in another P-r.